rooms-quickstart/README.md

```diff
@@ -0,0 +1,44 @@
+page_type: sample
+languages:
+- Python
+
+Products:
+- azure
+- azure-communication-rooms
+---
+
+
+# Manage Rooms
+
+For full instructions on how to build this code sample from scratch, look at [Quickstart: Create a room]
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+- Install [Python](https://www.python.org/downloads/) 3.6+.
+- Create an Azure Communication Services resource. For details, see [Quickstart: Create and manage Communication Services resources](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You'll need to record your resource endpoint for this quickstart.
+- Create a  [user access token](https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-python). Be sure to set the scope to **voip**, and note the **token** string as well as the **userId** string.
+
+## Code Structure
+
+- **./rooms-python-quickstart/rooms.py:** contains code for managing rooms.
+
+## Install the packages
+
+From a console prompt, navigate to the directory containing the rooms.py file, then execute the following command:
+
+- pip install azure_communication_rooms
+- pip install azure.communication.identity
```

Comment:
is this actually using dots here? from the communication identity sdk readme, it says install with this 
`pip install azure-communication-identity`